src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,128 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final double capacityIndex = 1.5;
+    private int size = 0;
+    private int capacity = 10;
+    private Object[] array;
```

Comment:
```suggestion
    private T[] values;
```
'array' is a bad naming